packages/context/package.json

```diff
@@ -8,12 +8,12 @@
     "build": "npm run build:lib && npm run build:lib6",
     "build:lib": "tsc --target es2017 --outDir lib",
     "build:lib6": "tsc --target es2015 --outDir lib6",
-    "clean": "rm -f loopback-context*.tgz && rm -rf lib*",
+    "clean": "rm -f loopback-context*.tgz && rm -rf lib* && rm -rf package",
```

Comment:
FWIW, `rm` accepts multiple file-path arguments, there is no need to call it multiple times.

```
rm -rf loopback-context*.tgz lib* package
```